packages/utils-tests-helpers/src/hint-runner.ts

```diff
@@ -244,6 +249,14 @@ export const testHint = (hintId: string, hintTests: HintTest[], configs: { [key:
         } catch (e) {
             console.error(e);
 
+            if (server) {
+                server.stop();
+            }
+
+            if (engine) {
+                await engine.close();
+            }
+
             return t.fail(`${hintTest.name} throwed an exception:\n${e.message}\n${e.stack}`);
```

Comment:
Should some version of `stopConnector` also be called here? Not sure if `hintTest.after` ever performs cleanup, but if it does it won't get run.

If so, perhaps we should just move all of the cleanup to a `finally` block.

Reply:
Not super concerned about not calling `hintTest.after`. The test has already failed so if there's ever a `test.after` action it doesn't matter that much.

Reply:
ok